### PR TITLE
Support for Wagtail 2.11 release

### PR DESCRIPTION
**Pull request by Stormheg**
Hi :wave: 

This PR adds support for the latest Wagtail 2.11 release. 

A new 1.4.0 release of this package should be made to PyPi.

### CODE REVIEW

**Comment by Stormheg** [comment]
Regarding the failing CI:

> coveralls.exception.CoverallsException: Running on Github Actions but GITHUB_TOKEN is not set. Add "env: GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}" to your step config.


Looks like Coveralls now expects to be passed `GITHUB_TOKEN`. I'll try adding that and see if that works...

**Comment by Stormheg** [comment]
That didn't work...

> coveralls.exception.CoverallsException: Could not submit coverage: 500 Server Error: Internal Server Error for url: https://coveralls.io/api/v1/jobs

**Comment by willbarton** [comment]
@Stormheg I'm trying to see what might be going on with Coveralls here.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -72,3 +72,4 @@ jobs:
         env: 
           TOXENV: ${{ matrix.toxenv }}
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}"
```
Comment (willbarton) [inline .github/workflows/test.yml:75]: I think there's an errant `"` here that's causing the 500 from Coveralls.
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Review by willbarton — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (willbarton) [reply .github/workflows/test.yml:75]: Once that's fixed, this PR looks good to me! I'll get it merged and a new version released.

Comment (Stormheg) [reply .github/workflows/test.yml:75]: > I think there's an errant `"` here that's causing the 500 from Coveralls.
> ```suggestion
>           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
> ```
> 

🤦‍♂️

Thanks for pointing that out!

**Comment by Stormheg** [comment]
@willbarton thanks, coveralls is working now!

**Comment by willbarton** [comment]
Thanks, @Stormheg!

**Pull request merged**